src/test/java/core/basesyntax/StorageTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class StorageTest {
+
+    @Test
+    void setFruitQuantity_validData_ok() {
+        Storage storage = new Storage();
+        storage.setFruitQuantity("banana", 100);
+        Assertions.assertEquals(100, storage.getAllFruits().get("banana"),
```

Comment:
if you test the store's setFruitQuantity method, don't use another store method to test the result. You can get the map directly making it public (check other tests)

Reply:
public field? Are you serious? I prefer to use getter: getAllFruits()

Reply:
There is nothing wrong with making the storage public because you are adding data to the storage using StorageDao. If you want private storage, you still need to add getters/setters, so encapsulation is still missing. In this test, you want to test the `setFruitQuantity()` method, but the result of the test depends on the implementation of the `getAllFruits()` method

Reply:
Even if I wanted to make this field public, I wouldn't be able to, because even the plugin wouldn't allow it - and rightly so